### DISCUSSION TITLE
Better performance with older versions of YOLO

### DISCUSSION THREAD

**Discussion by Josephts1** [ultralytics/hub#996] [Q&A]
Hi everyone.
I'm looking for the best YOLO model for detecting mandarin oranges. I'm using my own dataset with 236 images (196 for train, 30 for val and 10 for test).
Without any pre-processing or alteration of other parameters in the YOLO model result=model.train(data="/content/drive/MyDrive/Proyecto_de_grado/data/data.yaml", epochs=100,patience=50,batch=16,plots=True,optimizer="auto",lr0=1e-4,project="/content/drive/MyDrive/Proyecto_de_grado/runs/YOLO10/l")
I've performed a training for each version of YOLO, from the YOLOv8n version to the YOLO11l version (the model used was a new model, without pre-trained weights). I plotted the mAP50-95 performance metric in validation against the number of parameters for each version of YOLO and this was the result (see image 1).
My question is: Why do older versions of YOLO have better mAP50-95 val than newer versions of YOLO?

I attach the model validation code and an image of my data set (see image 2):
metrics = model.val(data='/content/drive/MyDrive/Proyecto_de_grado/data/data.yaml', project='/content/drive/MyDrive/Proyecto_de_grado/runs/YOLO10/l')

image 1
![image_2025-01-24_072618883](https://github.com/user-attachments/assets/eb7ce38b-7090-455c-babd-4130380dcfbf)

image 2
![Captura de pantalla 2025-01-24 065430](https://github.com/user-attachments/assets/814337a6-65b8-4f6a-a402-a7939d53b6f3)
